tests/integration/components/tasks/holder-test.js

```diff
@@ -199,41 +199,6 @@ module('Integration | Component | Tasks Holder', function (hooks) {
     @disabled={{this.disabled}}
   />`);
 
-    assert
-      .dom('[data-test-task-status-select]')
-      .hasValue(TASK_KEYS.IN_PROGRESS);
-
-    await select('[data-test-task-status-select]', TASK_KEYS.COMPLETED);
-
-    assert
-      .dom('[data-test-task-status-select]')
-      .hasValue(TASK_KEYS.IN_PROGRESS);
-  });
```

Comment:
why you are removing this test?

Reply:
https://github.com/Real-Dev-Squad/website-my/pull/522/files#diff-4edf657709000f49929d4181d55e9b1df7adce128255a72edc094ad86eed2168L180
https://github.com/Real-Dev-Squad/website-my/blob/develop/tests/integration/components/tasks/holder-test.js#L213

Please check these line